docs/validators/index.md

```diff
@@ -33,6 +33,25 @@ To have a **validator permit** in a given subnet, allowing you to submit miner e
 
 - You must be one of the top 64 validators in the subnet, ranked by stake weight.
 
+## Hotkey Association & Staking (subnet 0, the root subnet, only)
+
+:::tip Root Subnet (Subnet 0) only
+Skip this step if you are not registering a validator on the root subnet (subnet 0)
+:::
+
+```bash
+btcli wallet associate-hotkey --wallet.name  <wallet name> --hotkey <your hotkey>
+```
+Add stake to your validator before registering: 
+
+```bash
+# Stake funds to your hotkey account within the subnet.
+btcli stake add --wallet.name <wallet name> --wallet.hotkey <your validating hotkey>
+```
+:::tip Root Subnet (Subnet 0) only
```

Comment:
let's remove this tip, the info is already covered on the page. also I think the tip here shouldn't have subnet 0 in it, since you can't set weights on root subnet.